### DISCUSSION TITLE
Neutral voltage from Asymmetric/Three Phase Power Flow computation

### DISCUSSION THREAD

**Discussion by garciagimenezangel** [e2nIEE/pandapower#2151] [Q&A]
Hi, 
I'm a beginner in the world of electric engineer, and I'm using pandapower to run the Asymmetric/Three Phase Power Flow, in a Low-Voltage grid. I've seen that, after the power flow computation with runpp_3ph, the results stored at the buses (res_bus_3ph) include enough information to compute the voltage and power vectors as complex numbers, with its phasors, for the three phases. Similarly, the results stored at the line level (res_line_3ph) also includes the current values for the three phases, which combined with the power factor allows the description of the phases' current as vectors too. Nevertheless, I'm interested in obtaining the current and voltage, with their phasors, in the neutral. Afaik, the current intensity in the neutral can be computed as the summatory of the current vectors in the three phases, but how to find out the value of the voltage vector in the neutral? I'm assuming that voltages provided by pandapower are always referenced to a local voltage reference, and therefore the neutral can also suffer from voltage changes across the grid.
Thanks a lot for any help received, Cheers

**Comment by andreabragantini** [comment]
I'm also following this discussion, if anyone wants to clarify....
